src/rtFileDownloader.cpp

```diff
@@ -76,6 +76,7 @@ struct MemoryStruct
   char* headerBuffer;
   size_t contentsSize;
   char* contentsBuffer;
+  rtFileDownloadRequest *rtFileDownloadReq;
```

Comment:
Let's rename rtFileDownloadReq to downloadRequest to keep with the naming convention found in the other sources.  Also, can you please initialize this variable to NULL in the constructor?